packages/dds/tree/src/util/testChangeRebaser.ts

```diff
@@ -0,0 +1,165 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ChangeRebaser } from "../rebase";
+
+export type Passed = "Passed";
+
+export interface OutputType<TChange> {
+    /**
+     * "Passed" iff `(A ○ B) ○ C = A (B ○ C) = A ○ B ○ C`,
+     * otherwise a triple that violates the axiom.
+     */
+    composeAssociativity: Passed | [TChange, TChange, TChange];
+    /**
+     * "Passed" iff `A ↷ (B ○ C) = (A ↷ B) ↷ C`,
+     * otherwise a triple that violates the axiom.
+     */
+    rebaseLeftDistributivity: Passed | [TChange, TChange, TChange];
+    /**
+     * "Passed" iff `(A ○ B) ↷ C = (A ↷ C) ○ (B ↷ (A⁻¹ ○ C ○ (A ↷ C)) ↷ C)`,
+     * otherwise a triple that violates the axiom.
+     */
+    rebaseRightDistributivity: Passed | [TChange, TChange, TChange];
+    /**
+     * "Passed" iff `(A ↷ B) ↷ B⁻¹ = A`,
+     * otherwise a pair that violates the axiom.
+     */
+    rebaseOverDoUndoPairIsNoOp: Passed | [TChange, TChange];
+    /**
+     * "Passed" iff `((A ↷ B) ↷ B⁻¹) ↷ B = A ↷ B`,
+     * otherwise a pair that violates the axiom.
+     */
+    rebaseOverUndoRedoPairIsNoOp: Passed | [TChange, TChange];
+    /**
+     * "Passed" iff `A ○ A⁻¹ = ε` where `ε` is the empty change,
+     * otherwise a change that violates the axiom.
+     */
+    composeWithInverseIsNoOp: Passed | TChange;
+}
+
+/**
+ * Verifies the axioms of `ChangeRebaser` are met by the given `rebaser`.
+ * @param rebaser - The rebaser to test.
+ * @param changes - The set of changes to use for testing the `rebaser`.
+ * @param isEquivalent - Used to compare whether two changes are equivalent for the purposes of this axioms.
+ */
+export function verifyChangeRebaser<TChange>(
```

Comment:
Don't forget to update the file name (and the test file name) to match the new name of the function.